### PR TITLE
added support to parse scope from DefaultKey and add it as a prefix f…

### PR DESCRIPTION
**Pull request by EyalHouzz**
…or consumer/pro

### CODE REVIEW

**Comment by EyalHouzz** [comment]
@tomerhouzz / @TamirTwina  do you know how to trigger the new build after this change is merged?

File: Xcode-Extension/DefaultsCommand.swift [hunk 1]
```diff
@@ -114,22 +121,27 @@ struct DefaultKey {
...
         self.default = value
         self.key = key
+        self.scope = scope
     }
 }
 
 
 extension DefaultKey {
     var prefKey: String {
+        let prefix = scope != .common ? (scope.rawValue + ".") : ""
```
Comment (GermanVelibekovHouzz) [inline Xcode-Extension/DefaultsCommand.swift:139]: Isn't `prefix` a Swift literal?

**Comment by tomerhouzz** [comment]
@EyalHouzz you need to upload an archive to github releases as a new release. DM me if you find it doesn't work for you

File: Xcode-Extension/DefaultsCommand.swift (hunk 1 above)
Comment (EyalHouzz) [reply Xcode-Extension/DefaultsCommand.swift:139]: @GermanVelibekovHouzz it compiled and tested so I guess it's ok

**Pull request merged**